### ISSUE TITLE
Update documentation to fix disjointed Description and Details sections

### ISSUE THREAD

**Issue by mattwarkentin** [mlverse/torchvision#16]
Hi @dfalbel,

Excellent work on `torch` and `torchvision`! This is an exciting development for the `R` community and clearly an immense amount of work has been put into it.

I have noticed that for both `torch` and `torchvision` that you have generated your `roxygen` tags as follows (using `folder-dataset.R` as an example):

```r
#' Folder dataset
#'
#' A generic data loader where the images are arranged in this way:
#'
#' ```
#' root/dog/xxx.png
#' root/dog/xxy.png
#' root/dog/xxz.png
#'
#' root/cat/123.png
#' root/cat/nsdf3.png
#' root/cat/asd932_.png
#' ```
#'
#' @param root Root directory path.
#' @param loader A function to load an image given its path.
#' @param transform  A function/transform that takes in an PIL image and returns
#'   a transformed version. E.g, [transform_random_crop()].
#' @param target_transform A function/transform that takes in the target and
#'   transforms it.
#' @param is_valid_file A function that takes path of an Image file and check if
#'   the file is a valid file (used to check of corrupt files)
#'
#' @family datasets
#'
#' @importFrom torch dataset
#' @export
image_folder_dataset <- dataset(
  "image_folder",
  inherit = folder_dataset,
  initialize = function(root, transform=NULL, target_transform=NULL,
                        loader=NULL, is_valid_file=NULL) {

    if (is.null(loader))
      loader <- magick_loader

    if (!is.null(is_valid_file))
      extensions <- NULL
    else
      extensions <- IMG_EXTENSIONS

    super$initialize(root, loader, extensions, transform=transform,
                     target_transform=target_transform,
                     is_valid_file=is_valid_file)
    self$imgs <- self$samples
  }
)
```

The first three sections are treated as `@title`, `@description`, and `@details` respectively, and while it looks fine in the `R` script, when the resulting documentation is viewed in `R`/`RStudio` it separates the `Description` and `Details` section in a way that makes the documentation look incomplete:
![Screen Shot 2020-09-29 at 12 54 03 PM](https://user-images.githubusercontent.com/27825069/94589134-e5bee980-0252-11eb-9e06-1a69b8166ce2.png)

I would be happy to draft up a PR which updates the documentation to fix these issues, if you would be interested in me doing this. I just wanted to check with you first before getting into it.

**Comment by dfalbel** [comment]
Hi @mattwarkentin 

I didn't notice this was happening :( 
A PR fixing these is definitely welcome and would be really helpful.

For torchvision we are using the default `devtools::document()` workflow to generate the docs, but for torch we use a [custom script](https://github.com/mlverse/torch/blob/master/tools/document.R) to automate the `if (torch_installed())` in the examples.

**Comment by mattwarkentin** [comment]
Sounds good! I will start working on a PR for `torchvision` and we can go from there.

**Comment by mattwarkentin** [comment]
Closed with #17
